troposphere/static/js/components/images/detail/versions/Version.react.js

```diff
@@ -21,29 +26,42 @@ export default React.createClass({
         showAvailability: React.PropTypes.bool
     },
 
-    getDefaultProps: function() {
+    getDefaultProps() {
         return {
             showAvailability: true,
             editable: true
         }
     },
 
-    onEditClicked: function() {
+    onCardClick() {
+        let isOpen = this.state.isOpen
+            ? false : true;
```

Comment:
OK, I like it. `let isOpen = !this.state.isOpen;` is better as long as people can tell what's happening. 
Thank you